exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -689,7 +702,7 @@ open class Table(name: String = "") : ColumnSet(), DdlAware {
 
     /** Creates a numeric column, with the specified [name], for storing 1-byte integers. */
     fun byte(name: String): Column<Byte> = registerColumn(name, ByteColumnType()).apply {
-        check("${generatedSignedCheckPrefix}byte_$name") { it.between(Byte.MIN_VALUE, Byte.MAX_VALUE) }
+        check("${generatedSignedCheckPrefix}byte_${this.unquotedName()}") { it.between(Byte.MIN_VALUE, Byte.MAX_VALUE) }
```

Comment:
Are we using unquoted column names because it was broken before (so it wouldn't work if name was quoted before)?

Because, if it wasn't, then technically that means we've changed the expected/working naming convention for our defined check constraints.

It probably won't impact many users, but it'd be better to play safe and include a note that implicit check constraint naming convention has changed. In the event users have, for example, been retrieving and storing constraint names to be used elsewhere (like in plain sql etc).

Reply:
> Are we using unquoted column names because it was broken before (so it wouldn't work if name was quoted before)?

Yes, it was broken before for quoted names.